templates/repo/issue/view_content/comments.tmpl

```diff
@@ -275,7 +304,8 @@
 				<img src="{{.Poster.RelAvatarLink}}">
 			</a>
 			<span class="text grey">
-				{{$.i18n.Tr "repo.issues.dependency.added_dependency" .Poster.HomeLink (.Poster.GetDisplayName|Escape) $createdStr | Safe}}
+				<a class="author" href="{{.Poster.HomeLink}}">{{.Poster.GetDisplayName}}</a>
+				{{$.i18n.Tr "repo.issues.dependency.added_dependency" $createdStr | Safe}}
```

Comment:
```suggestion
				{{$.i18n.Tr "repo.issues.dependency.added_dependency" .Poster.HomeLink (.Poster.GetDisplayName|Escape) $createdStr | Safe}}
```
To maintain compatibility; this should not affect your new layout since you'd be using just `%[3]s`.